tests/e2e_logging.py

```diff
@@ -81,8 +81,7 @@ def run(args):
     args.package = "libloggingenc"
     notify_server_host = "localhost"
     args.notify_server = (
-        notify_server_host
-        + ":"
+        notify_server_host   + ":"
```

Comment:
This is a deliberate format break to check that this actually produces a quickly failing build, will remove once that's run.

Reply:
Yup:

```
static_checks
Failed

ACC_1804_SGX_build
Skipped

ACC_1804_virtual_coverage_build
Skipped

ACC_1804_virtual_SAN_build
Skipped

ACC_1804_SGX_perf_build
Skipped
```

And
```
Duration:
0:22.368
```